include/conversation.php

```diff
@@ -394,10 +394,9 @@ function visible_activity($item) {
 		}
 	}
 
-	if (activity_match($item['verb'], ACTIVITY_FOLLOW) && $item['object-type'] === ACTIVITY_OBJ_NOTE) {
-		if (!($item['self'] && ($item['uid'] == local_user()))) {
-			return false;
-		}
+	/// @TODO such things are unneccessary code-complexibilities and should be avoided
```

Comment:
I don't think this TODO has its place here, please suggest an improvement for readability (new lines in the conditions or an explanation of the incredibly specific condition) instead.

Reply:
Where should I do this then? !developers group? I would like to have the `if()` block gone ands replaced with 2 lines: `$isVisible = (allConditionsHere);` and then `return $isVisible;`.